### DISCUSSION TITLE
Pertanyaan Q5: Time Series & Forecasting no 8

### DISCUSSION THREAD

**Discussion by Zahraa02** [teamalgoritma/community#930] [Q&A] [answered]
![Q5](https://github.com/user-attachments/assets/1a527509-4ae7-469e-a791-aa0f60be2eb1)
Saya agak bingung, kesalahan saya di soal ini ada dimana yaa? Saya kira karena hasil p-value < alpha, maka residual/ errornya berkorelasi atau memiliki pola, apakah pemahaman itu salah?

**Comment by IrfanChairurrachman** [comment]
Dear bu @Zahraa02 ,

Untuk pemahaman Ljung-Box telah tepat, ketika p-value < 0.05 maka terdapat autokorelasi. Hanya saja dalam hal ini mungkin terdapat perbedaan cara ketika membuat model arimanya Bu.

Boleh minta tolong untuk di screenshotkan chunk ketika melakukan cross-validation dan pembuatan model Arimanya. 

Sekian, ditunggu jawabannya, terima kasih.

**Reply by Zahraa02** [reply]
![image](https://github.com/user-attachments/assets/e50ce02f-b6c6-40b0-a625-a1b4862bcdd2)
![image](https://github.com/user-attachments/assets/6b4958a5-5899-45b7-b973-202a3692730e)

**Reply by IrfanChairurrachman** [reply, marked as answer]
dear bu @Zahraa02 ,

Sepertinya hal ini terjadi karena perbedaan karakteristik model ARIMAnya. Ketika membuat model auto.arima() sebaiknya tidak memberikan parameter `d=0` berhubung auto.arima akan menghitung secara otomatis d yang dibutuhkan oleh data kita, hasilnya akan cukup berbeda berhubung auto.arima menggunakan uji stationer lain selain `adf.test()`, yaitu KPSS test. 

Jika tidak menggunakan `d=0` maka hasil box ljung test nya juga akan berbeda Bu.

Sekian jawaban dari kami, semoga membantu

**Reply by Zahraa02** [reply]
ooh baik, setelah saya coba ulang dengan menghilangkan parameter d=0 di auto.arima, hasil Ljung- Box Testnya juga berbeda. Terima kasih mas atas bantuannya.

**Reply by IrfanChairurrachman** [reply]
Baik, sama-sama Bu
